e2e/scenario_test.go

```diff
@@ -984,6 +984,74 @@ func Test_ubuntu2204Wasm(t *testing.T) {
 	})
 }
 
+func Test_Ubuntu2204DisableKubeletServingCertificateRotationWithTags(t *testing.T) {
```

Comment:
do we want to add a test for when the flag isnt set but the nodepool tag is passed and verify the behavior? to account for any funky commands a cx may pass.

Reply:
added